suite-native/atoms/src/Card.tsx

```diff
@@ -10,7 +10,7 @@ type CardProps = {
 
 const cardStyle = prepareNativeStyle(utils => ({
     backgroundColor: utils.colors.gray0,
-    borderRadius: 12,
+    borderRadius: utils.borders.radii.large / 2,
```

Comment:
maybe we should consider redefining border radiuses in theme since we are constantly bending them like this... definitely not part of this PR but something that could be discussed with UI team because they have some consistent rule for it